settings/authentication-personalization/authentication-setup/mintlify.mdx

```diff
@@ -0,0 +1,35 @@
+---
+title: 'Mintlify Dashboard Handshake'
+description: 'Use a customized login flow to authenticate users'
+---
+
+<Info>
+  This is the documentation for the Mintlify Dashboard **Authentication** Handshake. The Mintlify Dashboard Handshake is not available for Personalization.
+</Info>
+
+If your documentation readers are also your documentation editors, you can allow Mintlify to manage access to your documentation. Anyone that can access
+your dashboard will automatically be able to access your documentation.
+
+## Implementation
+
+<Steps>
+  <Step title="Configure your Authentication settings">
+    Go to your [Mintlify dashboard settings](https://dashboard.mintlify.com/mintlify/mintlify/products/authentication) and enable the Mintlify Dashboard Handshake.
```

Comment:
Can we change all dashboard URLs so it removes the org and deployment? the way it is currently, it wont properly redirect the user

```suggestion
    Go to your [Mintlify dashboard settings](https://dashboard.mintlify.com/products/authentication) and enable the Mintlify Dashboard Handshake.
```

Reply:
Oh I thought it would - try clicking this link

https://dashboard.mintlify.com/definitely-not-a-real-org/foobar/products/authentication

Reply:
wtf - ricky spread misinformation

Reply:
ricky says it's still ugly when they hover over the link and that the links should be clean

Reply:
updated